easybuild/easyconfigs/g/g2clib/g2clib-1.7.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,32 @@
+name = 'g2clib'
+version = '1.7.0'
+easyblock = 'CMakeMake'
+
+homepage = 'https://github.com/NOAA-EMC/NCEPLIBS-g2c'
+description = """Library contains GRIB2 encoder/decoder ('C' version)."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/NOAA-EMC/NCEPLIBS-g2c/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['73afba9da382fed73ed8692d77fa037bb313280879cd4012a5e5697dccf55175']
+
+builddependencies = [('binutils', '2.38'),
+                     ('CMake', '3.23.1'),
+                     ]
+
+dependencies = [
+    ('JasPer', '2.0.33'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.1.3'),
+]
+
+parallel = 8
```

Comment:
is restricting parallelism still needed with the switch to `CMake`?
```suggestion
```

Reply:
I do it to not strangle systems. But fine, should be handled automatically

Reply:
@surak unless it causes build issues, parallelism constraints are best left to easybuild users, since sensible values will vary across different systems. you can always control what easybuild will use using the `EASYBUILD_PARALLEL=X` environment variable, `--parallel=X` command line option or corresponding EB configuration file parameter